core/tests/ts-integration/scripts/build.ts

```diff
@@ -0,0 +1,32 @@
+import path from 'path';
+import { exec } from 'child_process';
+import {
+    needsRecompilation,
+    setCompilationTime,
+    isFolderEmpty,
+    CONTRACTS_DIR,
+    OUTPUT_DIR,
+    TIMESTAMP_FILE
+} from './utils';
+
+async function main() {
```

Comment:
please add comment - describing what we're doing here (using some .timestamp file to record the last time that files were changed)

Reply:
+

Reply:
seems that it was not done.
I'm looking for a 2-line comment, where reader can quickly see what's special about this 'build' script, without having to read the details of the code.

Reply:
updated

Reply:
I'm not sure it's the right move. 
We could solve in two ways: 
1. Move l2 contracts compilation to foundry zksync 
2. Use the prebuild contracts from the repo. we do pulbish built contracts